app/server/emergency_app/controllers/emergency_controller.js

```diff
@@ -0,0 +1,61 @@
+const model = require("../models/emergency_model");
+
+
+
+
+
+
+/**
+ * This will return list of emergency support
+ */
+ const getEmergencyList = async (req, res) => {
+    try {
+        const result = await model.getEmergencyList();
+        res.status(200);
+        res.json(result);
+    } catch (err) {
+        res.status(500);
+        res.send(err.meassage);
```

Comment:
same here

Reply:
Ohh yeah it's obviously a typo that got carried over, will correct it asap